src/main/java/core/mate/academy/data/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.mate.academy.data;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```
enums should be in singular